main.py

```diff
@@ -0,0 +1,60 @@
+from typing import List
+
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from db.engine import SessionLocal
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get("/")
+async def root():
+    return {"message": "Hello World"}
+
+
+@app.get("/authors/", response_model=List[schemas.Author])
+def get_authors(db: Session = Depends(get_db)):
+    return crud.get_authors(db)
```

Comment:
The `get_authors` endpoint should support pagination by accepting `skip` and `limit` query parameters to retrieve a subset of authors. Consider adding these parameters and using them in the query.